packages/fex/build.sh

```diff
@@ -0,0 +1,36 @@
+TERMUX_PKG_HOMEPAGE=https://wiki.fex-emu.org/index.php/Main_Page
+TERMUX_PKG_DESCRIPTION="Fast x86 emulation frontend"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=2201
+TERMUX_PKG_SRCURL=https://github.com/FEX-Emu/FEX.git
+TERMUX_PKG_GIT_BRANCH=FEX-${TERMUX_PKG_VERSION}
+TERMUX_PKG_DEPENDS="libandroid-shmem, libc++"
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
+-DBUILD_TESTS=OFF
+-DENABLE_LTO=OFF
+-DENABLE_JEMALLOC=OFF
```

Comment:
This option in particular is going to be a bit rough.
JEMalloc is required by FEX to wrap most 32-bit applications.
Someone trying to run Steam for example will run out of virtual address space shortly in to the application booting.